lib/geocode.js

```diff
@@ -53,11 +52,30 @@ function _handleArcGISResponse (error, response, body, callback) {
  * geoservice.geocode({ text: "920 SW 3rd Ave, Portland, OR 97204" }, callback);
 */
 function geocode (parameters, callback) {
-  parameters.f = parameters.f || "json";
```

Comment:
Oh, it was that change from this to the more explicit `if (!parameters.f) {}` that I was wondering about. I get what it's doing and why, just wondering why the change.